Python/errors.c

```diff
@@ -1287,12 +1287,13 @@ write_unraisable_exc_file(PyThreadState *tstate, PyObject *exc_type,
     }
 
     assert(PyExceptionClass_Check(exc_type));
-    const char *className = PyExceptionClass_Name(exc_type);
-    if (className != NULL) {
-        const char *dot = strrchr(className, '.');
-        if (dot != NULL) {
-            className = dot+1;
-        }
+    PyObject *qualName = PyType_GetQualName((PyTypeObject *)exc_type);
+    if (!qualName) {
```

Comment:
Nit: suggest using explicit `NULL` checks for consistency with the majority of the code base. Ditto for the other `NULL` checks.

```suggestion
    if (qualName != NULL) {
```

Also, snake case is preferred for local variables.

Reply:
Thanks.


> Also, snake case is preferred for local variables.

There were already camel case className and moduleName, so I made it match. I can change all of them but it will make the diff noisier so let's get the content reviewed first.

Reply:
I wouldn't change all of them; only the ones that are introduced. Just my personal preference.